src/components/createPostModal/index.js

```diff
@@ -33,6 +31,27 @@ const CreatePostModal = () => {
     }
 
     return (
+        <>
+            <section className="create-post-user-details">
+                <div className="profile-icon"><p>AJ</p></div>
+                <div className="post-user-name"><p>Alex J</p></div>
```

Comment:
Shouldn't there be a variable with the user's name? Why are we hard coding Alex J?